source/import-export.txt

```diff
@@ -158,6 +162,47 @@ Procedure
       .. include:: /includes/steps/export-filter.rst
 
       .. include:: /includes/fact-export-progress-bar.rst
+   
+   .. tab:: Export Aggregation Results
+      :tabid: export-Aggregation
+
+      To export results from your aggregation pipeline:
+
+      .. procedure::
+         :style: normal
+
+         .. step:: Connect to the deployment containing the
```

Comment:
For some reason this step title isn't showing as bold, but the rest of the steps are. Can you try putting the rst for this step all on one line and see if that fixes it? We'll go over the 72 char limit, but it might be necessary for formatting consistency here.